supporter-product-data/cloudformation/dynamo-tables.yaml

```diff
@@ -0,0 +1,56 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Supporter product data tables - stack name should be supporter-product-data-tables-${Stage}
+Parameters:
+  Stage:
+    Description: Environment name ( CODE, UAT or PROD)
+    Type: String
+    AllowedValues:
+      - PROD
+      - UAT
+      - CODE
+    Default: CODE
+Mappings:
+  StageVariables:
+    CODE:
+      DynamoWriteCapacity: 1
+      DynamoReadCapacity: 5
+    UAT:
+      DynamoWriteCapacity: 1
+      DynamoReadCapacity: 5
+    PROD:
+      DynamoWriteCapacity: 10
+      DynamoReadCapacity: 200
```

Comment:
we can keep this under review for PROD

Reply:
Yes I've been tweaking it as I go along. That read capacity comes from the attribute database in members-data-api which I think is a fairly good guide, might be able to drop the write capacity though